11 SpinnerAsync/src/components/app.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
+import ReduxThunk from 'redux-thunk';
```

Comment:
I'm not sure but It's possible. In the export it's as follows: `export default thunk;`.

And so, What do you think about change its name to thunk?:

`import thunk from 'redux-thunk';`